openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1173,6 +1173,7 @@ The following parameter MUST be used to communicate the specifics of the Credent
 This section defines the structure of the objects that appear in the `credentials_supported` metadata parameter.
 
 * `format`: REQUIRED. A JSON string identifying the format of this credential, i.e., `jwt_vc_json` or `ldp_vc`. Depending on the format value, the object contains further elements defining the type and (optionally) particular claims the credential MAY contain and information about how to display the credential. (#format_profiles) defines Credential Format Profiles introduced by this specification.
+* `credentials_supported_identifier`: REQUIRED when Credential Offer is used. A JSON string uniquely identifying this object within the `credentials_supported` array. It is used in the Credential Offer as defined in (#credential_offer_parameters) to enable the Wallet to discover the information about the Credential being offered.
```

Comment:
Since this is a unique identifier could we consider renaming this to `id` and moving it above `format` in this definition to signify the importance?

Reply:
Within VCI, there are two different identifiers: 1) identifier that points to a credentials_supported object in the issuer metadata, and 2) identifier that identifies a credential instance during issuance (PR #65), so I was trying to differentiate those.

Having said that, there is only one identifier within issuer metadata, so maybe just "id" is ok...

There are so many "id" claims in the Credential space: in the DIF presentation exchange, in W3C VCDM, that i would prefer differentiating from those, but maybe no need...

On the order, I usually like putting mandatory parameters at the top, but in this case, logically, identifier should probably be at the top.

Reply:
I agree with @cobward , it is the primary identifier and belongs to the top. I was also wondering why its not named `id` but I'm fine with the explanation

Reply:
with `credentials_subject` structure changing to a map, there will be no more `credentials_supported_identifier` :)
@cobward I think this data structure change of a `credentials_supported_identifier` is a breaking change for your implementation.. please check.